apps/token-manager/app/package.json

```diff
@@ -41,13 +41,14 @@
     "prettier": "^1.8.2"
   },
   "scripts": {
-    "test": "jest",
     "lint": "eslint ./src",
```

Comment:
Similar to the app's package.json, let's move the `lint` script to be just before `build`